src/database/models/person.js

```diff
@@ -0,0 +1,69 @@
+import { Model, DataTypes } from 'sequelize';
+import sequelize from '../db.js';
+
+class Person extends Model { }
+
```

Comment:
Why do you need this empty class?

Reply:
I need that class because it allows the init of the Person.init() function.
If I delete that class the function not work